src/java.desktop/share/classes/sun/print/PathGraphics.java

```diff
@@ -1132,6 +1133,9 @@ protected BufferedImage getBufferedImage(Image img) {
             // VI needs to make a new BI: this is unavoidable but
             // I don't expect VI's to be "huge" in any case.
             return ((VolatileImage)img).getSnapshot();
+        } else if (img instanceof MultiResolutionImage) {
+            return convertToBufferedImage((MultiResolutionImage) img,
+                                           img.getWidth(null), img.getHeight(null));
```

Comment:
Any particular reason of using getWidth/getHeight(null) as seems like in spec, this observer parameter is ignored so we can directly use getWIdth/getHeight, I presume...

Reply:
No, we have to pass null if the parameter is ignored. Without it we will get error.